utils.py

```diff
@@ -0,0 +1,41 @@
+import shutil
+import os
+import yaml
+
+def init_nl(args):
+    # Define the path to the template inference.py file
+    inference_path = os.path.join(os.path.dirname(__file__), 'templates', 'inference.py')
+    config_path = os.path.join(os.path.dirname(__file__), 'templates', 'config.yaml')
+    
+    # Get the current working directory
+    root_dir = os.getcwd()
+    
+    # Define the path where you want to copy the template file
+    destination_path_inference = os.path.join(root_dir, 'inference.py')
+    config_path_inference = os.path.join(root_dir, 'config.yaml')
+
+    # Copy the template file to the current directory
+    shutil.copyfile(inference_path, destination_path_inference)
+    shutil.copyfile(config_path, config_path_inference)
+
+    print("initialized neura-launch project")
+
+def add_token(args):
+    token = args.token
```

Comment:
Also, instead of taking the token through args, take it like a password, where it's not visible in the shell.

Reply:
I am wondering what others think of this @rohansingh9001 @kunxl-gg ... I believe this project-level token is not a very secret thing (or is it?) so I didn't care much about making it like a password.

Reply:
it is a secret. Any token should not be in code.

Reply:
@XanderWatson The modern line length limit is now 127. 72 was the old limit.

Reply:
Also yes, follow PEP8. The least one should do while making a python package.

To check for our rules use

`flake8 --max-line-length 127 .`

Reply:
yes, sory for not following PEP8 already! I'll fix that

Reply:
also about the project level token - I don't understand the issue with the current implementation. that looks very consistent to the existing patterns in other libraries...